src/modules/leavePage.ts

```diff
@@ -13,7 +22,7 @@ const leavePage = function(data, { popstate = false, skipTransition = false } =
 	}
 
 	// animation promise stuff
-	const animationPromises = this.getAnimationPromises('out');
+	const animationPromises: Promise<void>[] = this.getAnimationPromises();
```

Comment:
This is one of the places where code was actually updated, since TS showed the param was never used.

Reply:
The `in` and `out` string arguments should definitely be part of it. Swup itself doesn't use them but the JS plugin expects them to be there I think?

Reply:
Oh man, that must be it... I'll update it.

Reply:
Another great example how overwriting the Swup instance can be dangerous. 😄

Reply:
Haha, yeah, I've also pushed commits in the past removing that same argument but somehow remembered before merging...